scripts/docs/build-dev.sh

```diff
@@ -1,11 +1,12 @@
 #! /usr/bin/env bash
 
-set -ev # exit when error
+set -e # exit when error
```

Comment:
Just for information, what is `v` stand for?

Reply:
verbose, every line of the script will be also written on the output (not just the effect, but also the line itself).

doc: http://tldp.org/LDP/abs/html/options.html (not easy to find!)